docs/src/using.md

```diff
@@ -163,40 +163,40 @@ julia> y[1]
 CategoricalArrays.CategoricalString{UInt32} "Old" (3/3)
 ```
 
-Missing values can be introduced either manually, or by restricting the set of possible levels. Let us imagine this time that we actually do not know the age of the first individual. We can set it to a missing value this way:
+Missing values can be introduced either manually, or by restricting the set of possible levels. Let us imagine this time that we actually do not know the age of the first individual. We can set it to a missing value this way (note that `⯑` is the compact representation of `missing`):
 
 ```jldoctest using
-julia> y[1] = null
-null
+julia> y[1] = missing
+missing
 
 julia> y
-4-element CategoricalArrays.CategoricalArray{Union{Nulls.Null, String},1,UInt32}:
- null    
+4-element CategoricalArrays.CategoricalArray{Union{Missings.Missing, String},1,UInt32}:
+ ⯑
  "Young" 
  "Middle"
  "Young" 
 
 julia> y[1]
-null
+missing
 
 ```
 
-It is also possible to transform all values belonging to some levels into missing values, which gives the same result as above in the present case since we have only one individual in the `"Old"` group. Let's first restore the original value for the first element, and then set it to missing again using the `nullok` argument to `levels!`:
+It is also possible to transform all values belonging to some levels into missing values, which gives the same result as above in the present case since we have only one individual in the `"Old"` group. Let's first restore the original value for the first element, and then set it to missing again using the `missingok` argument to `levels!`:
 
 ```jldoctest using
 julia> y[1] = "Old"
 "Old"
 
 julia> y
-4-element CategoricalArrays.CategoricalArray{Union{Nulls.Null, String},1,UInt32}:
+4-element CategoricalArrays.CategoricalArray{Union{Missings.Missing, String},1,UInt32}:
  "Old"   
  "Young" 
  "Middle"
  "Young" 
 
-julia> levels!(y, ["Young", "Middle"]; nullok=true)
-4-element CategoricalArrays.CategoricalArray{Union{Nulls.Null, String},1,UInt32}:
- null    
+julia> levels!(y, ["Young", "Middle"]; missingok=true)
```

Comment:
We should find a better name for that argument, `nullok` wasn't great either. Maybe `allow_missing=true`?